src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -1800,7 +1812,7 @@ public SyntaxNode ObjectCreationExpression(SyntaxNode type, params SyntaxNode[]
         /// </summary>
         public SyntaxNode ObjectCreationExpression(ITypeSymbol type, params SyntaxNode[] arguments)
```

Comment:
we're making all sorts of disconnects between our overloads.  I would just have the 'addImport' overload for making types.  Then i would have an ObjectCreationExpression overload that takes a SyntaxNode for the type.